.changeset/fifty-schools-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/fe-fpm-writer': patch
+---
+
+Fix: writing into manifest.json always provides 2 spaces for tab size. Addition property 'tabInfo' can be passed in config for generatION functions or tab sizing will calculated before saving
```

Comment:
Addition ==> Additional
generatION ==> generation
tab sizing will BE calculated

Reply:
thanks, corrected in edc31bd4a0cb4a5059f18d900c6a747848d09ab4